### ISSUE TITLE
Project Extent | Implement extent definition using CTN segments

### ISSUE THREAD

**Issue by amenity** [cityofaustin/atd-data-tech#4892]

**Comment by mddilley** [comment]
Alan has created the [CTN vector tile layer](https://github.com/cityofaustin/atd-data-tech/issues/4893#issuecomment-761164004) so I will being work on this first thing next week.

**Comment by mddilley** [comment]
This layer has been added in [this PR](https://github.com/cityofaustin/atd-moped/pull/186). One bug was encountered where, when capturing features from the vector tile layer, the line in a feature splits into multiple features. This results in only part of a feature being stored in the DB depending on the location of a click within the feature on the front end.

I talked with Alan about it on 1/22, and he is reaching out to ArcGIS for support. Mateo and Sergio also brought up a good point that this could be a bug on the client side.

Currently troubleshooting and here are some possible explanations.
https://gis.stackexchange.com/questions/232757/mapbox-gl-js-splitting-tileset-polygons
https://blog.cyclemap.link/2020-01-25-tilebuffer/

@johnclary also suggested grabbing a feature server if we can't resolve the issue.

**Comment by mddilley** [comment]
After some troubleshooting, I went back to the basic example of [using 3rd party vector tile layers with Mapbox GL JS](https://docs.mapbox.com/mapbox-gl-js/example/third-party/) and noticed that the source details were passed along with `maxzoom` and `minzoom` properties. Turns out that `maxzoom` is particularly important for Mapbox GL to know about for resolving features out of a vector tile layer.

I passed the `maxLOD` from the CTN layer [metadata](https://tiles.arcgis.com/tiles/0L95CJ0VTaxqcmED/arcgis/rest/services/CTN_Project_Extent_Vector_Tiles/VectorTileServer) as the `maxzoom` prop of the Source component, and I was unable to reproduce any features splitting into multiple features with the same ID afterwards.
